library/src/scala/quoted/Quotes.scala

```diff
@@ -3604,6 +3679,9 @@ trait Quotes { self: runtime.QuoteUnpickler & runtime.QuoteMatching =>
       *
       *  Note: Also see reflect.let
       *
+      *  @param parent The owner of the /var/lazy val
```

Comment:
Is this supposed to be:
```suggestion
      *  @param parent The owner of the val/var/lazy val
```

Reply:
Fixed